low-level/framework/sspl_rabbitmq_reinit

```diff
@@ -270,9 +270,9 @@ def _create_vhost_if_necessary(virtual_host, product):
         vhosts = subprocess.check_output(
             [RABBITMQCTL, 'list_vhosts']
             ).decode("utf-8").split('\n')
-        assert vhosts[0] == 'Listing vhosts ...'
-        assert vhosts[-2] == '...done.'
-        assert vhosts[-1] == ''
+        # assert vhosts[0] == 'Listing vhosts ...'
```

Comment:
These assert statements checks output which is very much version specific. I don't know for what purpose we have added that. But, I don't see any role here.
assert vhosts[-2] == '...done.'
This assert checks whether it has 'done...' in the output. But, for this version, this done is not even present in the output